module-2/backend/week_two.md

```diff
@@ -4,38 +4,90 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 
 Note: When you're done, submit a PR. 
 
-1. At a high level, what is ActiveRecord? What does it do/allow you to do?
-2. What is a migration?
-3. How does a table relate to a model?
-4. What kind of methods are `belongs_to`, and `has_many`? (i.e. class or instance) Give an example.
-5. What do they allow you to do?
-6. What's the difference between agile workflow and waterfall method?
-7. What is the difference between `#new` and `#create`?
-8. At a basic level, what does cURL allow you to do?
-9. In a database that's holding students and teachers, what will be the relationship between students and teachers? Draw the schema diagram.
-10. Define foreign key, primary key, and schema.
-11. Describe the relationship between a foreign key on one table and a primary key on another table.
-12. What are the parts of an HTTP response?
-13. `Rack::Test` allows us to test our controllers in isolation. What are some of the methods it gives us to simulate the request/response cycle?
-14. Describe some techniques to make our Sinatra views more DRY. Give an example of when you would use these techniques.
+1. At a high level, what is ActiveRecord? What does it do/allow you to do?<br>
+*Active record is an object relational mapping tool for using Ruby on databases. It allows us to interact with and manipulate databases by wrapping the database tables in a ruby object, so we can create our own methods and interact with the information in these tables.*
+
+2. What is a migration?<br>
+*A migration is the instructions for how we want our database to be created and/or altered.*
+
+3. How does a table relate to a model?<br>
+*A model represents a table in our database, and you can then methods on a model to interact with the information in the table.* 
+
+4. What kind of methods are `belongs_to`, and `has_many`? (i.e. class or instance) Give an example.<br>
+*These are class methods.<br>
```

Comment:
Actually, instance methods. Think of it this way, if are going to use the methods that `belongs_to` gives us, you would need to have a specific *instance* of a car. For example:

```ruby
car = Car.find(1) # this returns a specific instance of Car, the one with an id of 1.
dealer_name = car.dealership.name # we can then call `dealership` on the specific instance.
```